diff -burN php-pear-PHP_Beautifier-0.0.9.orig/PHP_Beautifier-0.0.9/Beautifier/Batch/Output/DirectoryBz2.php php-pear-PHP_Beautifier-0.0.9/PHP_Beautifier-0.0.9/Beautifier/Batch/Output/DirectoryBz2.php
--- php-pear-PHP_Beautifier-0.0.9.orig/PHP_Beautifier-0.0.9/Beautifier/Batch/Output/DirectoryBz2.php	2004-12-05 12:13:30.436060952 +0100
+++ php-pear-PHP_Beautifier-0.0.9/PHP_Beautifier-0.0.9/Beautifier/Batch/Output/DirectoryBz2.php	2004-12-05 12:22:11.407861200 +0100
@@ -26,7 +26,7 @@
     /**
     * Include PHP_Beautifier_Batch_DirectoryTar
     */    
-    require_once ('DirectoryTar.php');
+    require_once ('PHP/Beautifier/Batch/Output/DirectoryTar.php');
     /**
     * PHP_Beautifier_Batch_FilesGz
     * Handle the batch process for multiple php files to one directory.
diff -burN php-pear-PHP_Beautifier-0.0.9.orig/PHP_Beautifier-0.0.9/Beautifier/Batch/Output/DirectoryGz.php php-pear-PHP_Beautifier-0.0.9/PHP_Beautifier-0.0.9/Beautifier/Batch/Output/DirectoryGz.php
--- php-pear-PHP_Beautifier-0.0.9.orig/PHP_Beautifier-0.0.9/Beautifier/Batch/Output/DirectoryGz.php	2004-12-05 12:13:30.437060800 +0100
+++ php-pear-PHP_Beautifier-0.0.9/PHP_Beautifier-0.0.9/Beautifier/Batch/Output/DirectoryGz.php	2004-12-05 12:22:27.911352288 +0100
@@ -26,7 +26,7 @@
     /**
     * Include PHP_Beautifier_Batch_DirectoryTar
     */    
-    require_once ('DirectoryTar.php');
+    require_once ('PHP/Beautifier/Batch/Output/DirectoryTar.php');
     /**
     * PHP_Beautifier_Batch_FilesGz
     * Handle the batch process for multiple php files to one directory.
diff -burN php-pear-PHP_Beautifier-0.0.9.orig/PHP_Beautifier-0.0.9/Beautifier/Batch/Output/FilesBz2.php php-pear-PHP_Beautifier-0.0.9/PHP_Beautifier-0.0.9/Beautifier/Batch/Output/FilesBz2.php
--- php-pear-PHP_Beautifier-0.0.9.orig/PHP_Beautifier-0.0.9/Beautifier/Batch/Output/FilesBz2.php	2004-12-05 12:13:30.438060648 +0100
+++ php-pear-PHP_Beautifier-0.0.9/PHP_Beautifier-0.0.9/Beautifier/Batch/Output/FilesBz2.php	2004-12-05 12:23:15.008192480 +0100
@@ -26,7 +26,7 @@
     /**
     * Include PHP_Beautifier_Batch_FilesGz
     */    
-    require_once 'FilesTar.php';
+    require_once 'PHP/Beautifier/Batch/Output/FilesTar.php';
     /**
     * PHP_Beautifier_Batch_FilesBz2
     * Handle the batch process for one/multiple php files to one tar bzip2 file
diff -burN php-pear-PHP_Beautifier-0.0.9.orig/PHP_Beautifier-0.0.9/Beautifier/Batch/Output/FilesGz.php php-pear-PHP_Beautifier-0.0.9/PHP_Beautifier-0.0.9/Beautifier/Batch/Output/FilesGz.php
--- php-pear-PHP_Beautifier-0.0.9.orig/PHP_Beautifier-0.0.9/Beautifier/Batch/Output/FilesGz.php	2004-12-05 12:13:30.438060648 +0100
+++ php-pear-PHP_Beautifier-0.0.9/PHP_Beautifier-0.0.9/Beautifier/Batch/Output/FilesGz.php	2004-12-05 12:23:45.060623816 +0100
@@ -26,7 +26,7 @@
     /**
     * Require FilesTar
     */
-    include_once 'FilesTar.php';    
+    include_once 'PHP/Beautifier/Batch/Output/FilesTar.php';    
     /**
     * PHP_Beautifier_Batch_FilesGz
     * Handle the batch process for one/multiple php files to one tar gzipped file
diff -burN php-pear-PHP_Beautifier-0.0.9.orig/PHP_Beautifier-0.0.9/Beautifier/Batch.php php-pear-PHP_Beautifier-0.0.9/PHP_Beautifier-0.0.9/Beautifier/Batch.php
--- php-pear-PHP_Beautifier-0.0.9.orig/PHP_Beautifier-0.0.9/Beautifier/Batch.php	2004-12-05 12:13:30.447059280 +0100
+++ php-pear-PHP_Beautifier-0.0.9/PHP_Beautifier-0.0.9/Beautifier/Batch.php	2004-12-05 12:19:20.555834656 +0100
@@ -28,8 +28,8 @@
     /**
     * Require PHP_Beautifier_Decorator
     */
-    require_once ('Decorator.php');
-    require_once ('Batch/Output.php');
+    require_once ('PHP/Beautifier/Decorator.php');
+    require_once ('PHP/Beautifier/Batch/Output.php');
     // ArrayNested->off();
     // ArrayNested->on();
     
diff -burN php-pear-PHP_Beautifier-0.0.9.orig/PHP_Beautifier-0.0.9/Beautifier/StreamWrapper.php php-pear-PHP_Beautifier-0.0.9/PHP_Beautifier-0.0.9/Beautifier/StreamWrapper.php
--- php-pear-PHP_Beautifier-0.0.9.orig/PHP_Beautifier-0.0.9/Beautifier/StreamWrapper.php	2004-12-05 12:13:30.455058064 +0100
+++ php-pear-PHP_Beautifier-0.0.9/PHP_Beautifier-0.0.9/Beautifier/StreamWrapper.php	2004-12-05 12:19:20.557834352 +0100
@@ -42,5 +42,5 @@
         function dir_rewinddir();
         function dir_closedir();
     }
-    require_once('StreamWrapper/Tarz.php');
+    require_once('PHP/Beautifier/StreamWrapper/Tarz.php');
 ?>
\ Brak znaku nowej linii na koñcu pliku
diff -burN php-pear-PHP_Beautifier-0.0.9.orig/PHP_Beautifier-0.0.9/Beautifier.php php-pear-PHP_Beautifier-0.0.9/PHP_Beautifier-0.0.9/Beautifier.php
--- php-pear-PHP_Beautifier-0.0.9.orig/PHP_Beautifier-0.0.9/Beautifier.php	2004-12-05 12:13:30.397066880 +0100
+++ php-pear-PHP_Beautifier-0.0.9/PHP_Beautifier-0.0.9/Beautifier.php	2004-12-05 12:19:20.560833896 +0100
@@ -30,15 +30,15 @@
     /**
     * Require PHP_Beautifier_Filter
     */
-    include_once ('Beautifier/Filter.php');
+    include_once ('PHP/Beautifier/Filter.php');
     /**
     * Require PHP_Beautifier_Filter_Default
     */
-    include_once ('Beautifier/Filter/Default.filter.php');
+    include_once ('PHP/Beautifier/Filter/Default.filter.php');
     /**
     * Require PHP_Beautifier_Common
     */
-    include_once ('Beautifier/Common.php');
+    include_once ('PHP/Beautifier/Common.php');
     /**
     * Require Log
     */
@@ -46,11 +46,11 @@
     /**
     * Require Exceptions
     */
-    include_once ('Beautifier/Exception.php');
+    include_once ('PHP/Beautifier/Exception.php');
     /**
     * Require StreamWrapper
     */
-    include_once ('Beautifier/StreamWrapper.php');
+    include_once ('PHP/Beautifier/StreamWrapper.php');
     /**
     * PHP_Beautifier
     *
