diff -burN a.orig/b/Beautifier/Batch/Output/DirectoryBz2.php a/b/Beautifier/Batch/Output/DirectoryBz2.php
--- a.orig/b/Beautifier/Batch/Output/DirectoryBz2.php	2005-02-07 09:17:43.047075360 +0100
+++ a/b/Beautifier/Batch/Output/DirectoryBz2.php	2005-02-07 09:18:31.882651224 +0100
@@ -23,7 +23,7 @@
     /**
     * Include PHP_Beautifier_Batch_DirectoryTar
     */    
-    require_once ('DirectoryTar.php');
+    require_once ('PHP/Beautifier/Batch/Output/DirectoryTar.php');
     /**
     * PHP_Beautifier_Batch_FilesGz
     *
diff -burN a.orig/b/Beautifier/Batch/Output/DirectoryGz.php a/b/Beautifier/Batch/Output/DirectoryGz.php
--- a.orig/b/Beautifier/Batch/Output/DirectoryGz.php	2005-02-07 09:17:43.047075360 +0100
+++ a/b/Beautifier/Batch/Output/DirectoryGz.php	2005-02-07 09:19:00.361321808 +0100
@@ -23,7 +23,7 @@
     /**
     * Include PHP_Beautifier_Batch_DirectoryTar
     */    
-    require_once ('DirectoryTar.php');
+    require_once ('PHP/Beautifier/Batch/Output/DirectoryTar.php');
     /**
     * PHP_Beautifier_Batch_FilesGz
     *
diff -burN a.orig/b/Beautifier/Batch/Output/FilesBz2.php a/b/Beautifier/Batch/Output/FilesBz2.php
--- a.orig/b/Beautifier/Batch/Output/FilesBz2.php	2005-02-07 09:17:43.048075208 +0100
+++ a/b/Beautifier/Batch/Output/FilesBz2.php	2005-02-07 09:19:31.575576520 +0100
@@ -23,7 +23,7 @@
     /**
     * Include PHP_Beautifier_Batch_FilesGz
     */    
-    require_once 'FilesTar.php';
+    require_once 'PHP/Beautifier/Batch/Output/FilesTar.php';
     /**
     * Handle the batch process for one/multiple php files to one tar bzip2 file
     *
diff -burN a.orig/b/Beautifier/Batch/Output/FilesGz.php a/b/Beautifier/Batch/Output/FilesGz.php
--- a.orig/b/Beautifier/Batch/Output/FilesGz.php	2005-02-07 09:17:43.048075208 +0100
+++ a/b/Beautifier/Batch/Output/FilesGz.php	2005-02-07 09:19:46.708276000 +0100
@@ -23,7 +23,7 @@
     /**
     * Include PHP_Beautifier_Batch_FilesGz
     */    
-    require_once 'FilesTar.php';
+    require_once 'PHP/Beautifier/Batch/Output/FilesTar.php';
     /**
     * Handle the batch process for one/multiple php files to one tar gzip file
     *
diff -burN a.orig/b/Beautifier/Batch.php a/b/Beautifier/Batch.php
--- a.orig/b/Beautifier/Batch.php	2005-02-07 09:17:43.051074752 +0100
+++ a/b/Beautifier/Batch.php	2005-02-07 09:20:45.298368952 +0100
@@ -23,11 +23,11 @@
     /**
     * Require PHP_Beautifier_Decorator
     */
-    require_once ('Decorator.php');
+    require_once ('PHP/Beautifier/Decorator.php');
     /**
     * Require PHP_Beautifier_Batch_Output
     */
-    require_once ('Batch/Output.php');
+    require_once ('PHP/Beautifier/Batch/Output.php');
     // ArrayNested->off();
     // ArrayNested->on();
     
diff -burN a.orig/b/Beautifier/StreamWrapper.php a/b/Beautifier/StreamWrapper.php
--- a.orig/b/Beautifier/StreamWrapper.php	2005-02-07 09:17:43.053074448 +0100
+++ a/b/Beautifier/StreamWrapper.php	2005-02-07 09:21:05.335322872 +0100
@@ -51,5 +51,5 @@
         function dir_rewinddir();
         function dir_closedir();
     }
-    require_once('StreamWrapper/Tarz.php');
+    require_once('PHP/Beautifier/StreamWrapper/Tarz.php');
 ?>
\ Brak znaku nowej linii na koñcu pliku
diff -burN a.orig/b/Beautifier.php a/b/Beautifier.php
--- a.orig/b/Beautifier.php	2005-02-07 09:17:43.023079008 +0100
+++ a/b/Beautifier.php	2005-02-07 09:21:35.220779592 +0100
@@ -28,15 +28,15 @@
     /**
     * Require PHP_Beautifier_Filter
     */
-    include_once ('Beautifier/Filter.php');
+    include_once ('PHP/Beautifier/Filter.php');
     /**
     * Require PHP_Beautifier_Filter_Default
     */
-    include_once ('Beautifier/Filter/Default.filter.php');
+    include_once ('PHP/Beautifier/Filter/Default.filter.php');
     /**
     * Require PHP_Beautifier_Common
     */
-    include_once ('Beautifier/Common.php');
+    include_once ('PHP/Beautifier/Common.php');
     /**
     * Require Log
     */
@@ -44,11 +44,11 @@
     /**
     * Require Exceptions
     */
-    include_once ('Beautifier/Exception.php');
+    include_once ('PHP/Beautifier/Exception.php');
     /**
     * Require StreamWrapper
     */
-    include_once ('Beautifier/StreamWrapper.php');
+    include_once ('PHP/Beautifier/StreamWrapper.php');
     /**
     * PHP_Beautifier
     *
