--- PHP_Beautifier-0.1.11/usr/share/pear/PHP/Beautifier.php~	2007-01-19 13:18:00.190033925 +0200
+++ PHP_Beautifier-0.1.11/usr/share/pear/PHP/Beautifier.php	2007-01-19 13:20:39.363592430 +0200
@@ -27,15 +27,15 @@
 /**
  * Require PHP_Beautifier_Filter
  */
-include_once ('Beautifier/Filter.php');
+include_once ('PHP/Beautifier/Filter.php');
 /**
  * Require PHP_Beautifier_Filter_Default
  */
-include_once ('Beautifier/Filter/Default.filter.php');
+include_once ('PHP/Beautifier/Filter/Default.filter.php');
 /**
  * Require PHP_Beautifier_Common
  */
-include_once ('Beautifier/Common.php');
+include_once ('PHP/Beautifier/Common.php');
 /**
  * Require Log
  */
@@ -43,11 +43,11 @@
 /**
  * Require Exceptions
  */
-include_once ('Beautifier/Exception.php');
+include_once ('PHP/Beautifier/Exception.php');
 /**
  * Require StreamWrapper
  */
-include_once ('Beautifier/StreamWrapper.php');
+include_once ('PHP/Beautifier/StreamWrapper.php');
 /**
  * PHP_Beautifier
  *
